src/crypto/elliptic_r1.cpp

```diff
@@ -16,7 +16,7 @@ namespace fc { namespace crypto { namespace r1 {
           public_key_impl()
           :_key(nullptr)
           {
-          static int init = init_openssl();
+            init_openssl();
```

Comment:
This does not look correct. The static int would make sure it is only called once.

Reply:
There's already a static in init_openssl() to ensure it only to be called once.

// in file openssl.cpp:71
int init_openssl()
{
      static openssl_scope ossl;
      return 0;
}

Reply:
Excellent. thanks.